### PR TITLE
Truncated normal quantile

### LINKED ISSUE #1120: No implementation of quantile method for TruncatedNormal
Would it be possible to implement?

### PR DESCRIPTION
**Pull request by harahu**
- Add a naive implementation of the truncated normal quantile function. Fixes #1120
- Fix some dead links in the style guide.

### CODE REVIEW

**Comment by harahu** [comment]
@srvasude I followed your advice in #1120, with some help from [stats.stackexchange](https://stats.stackexchange.com/questions/490877/what-is-the-ppf-of-the-truncated-normal-distribution/491045?noredirect=1#comment907020_491045).

As mentioned in the `#TODO` I also added, this implementation might not be optimal from the perspective of numerical stability, but I feel it adds value over it not being implemented.

File: STYLE_GUIDE.md [hunk 1]
```diff
@@ -6,10 +6,11 @@ number of errors, and promote consistency.
 
 ## TensorFlow Style
 
-Follow the [TensorFlow style
-guide](https://www.tensorflow.org/community/style_guide) and [documentation
-guide](https://www.tensorflow.org/community/documentation), and in particular
-the [TensorFlow code
+Follow the [TensorFlow style
```
Comment (srvasude) [inline STYLE_GUIDE.md:9]: Revert changes in this file?

File: tensorflow_probability/python/distributions/truncated_normal.py [hunk 2]
```diff
@@ -420,6 +420,19 @@ def _mode(self):
     bc_loc = tf.broadcast_to(loc, shp)
     return tf.clip_by_value(bc_loc, low, high)
 
+  def _quantile(self, p):
+    # TODO: This implementation is analytically correct, but might not
+    #  perform well in all cases. See
+    #  https://en.wikipedia.org/wiki/Truncated_normal_distribution#Generating_values_from_the_truncated_normal_distribution
+    #  for a discussion on alternatives.
+    loc, scale, low, high = self._loc_scale_low_high()
+    alpha = (low - loc) / scale
```
Comment (srvasude) [inline tensorflow_probability/python/distributions/truncated_normal.py:429]: We generally avoid greek for variable names. How about `std_low` and `std_high`?

File: STYLE_GUIDE.md (hunk 1 above)
Comment (harahu) [reply STYLE_GUIDE.md:9]: Are you sure? The old links were dead. These seem to be working, at least the last time I checked.

File: tensorflow_probability/python/distributions/truncated_normal.py (hunk 2 above)
Comment (harahu) [reply tensorflow_probability/python/distributions/truncated_normal.py:429]: I'd love some clearer names for these, but I am unsure what would be appropriate. These are the same alpha and beta that are defined here: https://en.wikipedia.org/wiki/Truncated_normal_distribution

To me they just seem like convenient placeholders allowing for a more compact expression. What is your motivation for `std_low` and `std_high` in particular?

Comment (srvasude) [reply tensorflow_probability/python/distributions/truncated_normal.py:429]: `std_low` and `std_high` are used elsewhere in this file for this expression. It is also expressing that these are standard scores for `low` and `high`.

Comment (harahu) [reply tensorflow_probability/python/distributions/truncated_normal.py]: I renamed the variables given the naming scheme you suggested.

File: STYLE_GUIDE.md (hunk 1 above)
Comment (harahu) [reply STYLE_GUIDE.md:9]: Or would you want this in a separate PR? Can do that too.

Comment (ColCarroll) [reply STYLE_GUIDE.md:9]: I think just neater if this is in a separate PR (thanks for catching this!)

Comment (harahu) [reply STYLE_GUIDE.md]: I dropped these changes from this PR. Will make a separate one for these.

Comment (harahu) [reply STYLE_GUIDE.md]: New PR: https://github.com/tensorflow/probability/pull/1332

**Comment by harahu** [comment]
Looks like the linting failure isn't caused by my changes.

**Review by ColCarroll — APPROVED** [review]
Looks good -- thank you!

**Pull request merged**